File: CodeForWin/Loop/Exercise 28/Solution.py
#!/bin/bash/env python3

def checkPrime(number):
    for x in range(1, number + 1, 1):
        if number % x == 0:
            return 1


if __name__ == "__main__":

    limit = input("[+] Input upper limit: ")

    sum = 0
    primeNumbers = []

    for x in range(1, int(limit)+1):
        for i in range(2, x):
            if x % i == 0:
                print(x)
                sum += x

    print("Sum of prime numbers between 1-%d: " % int(limit))
    for y in primeNumbers:
        print(y, end = ", ")

File: CodeForWin/Loop/Exercise 28/Solution.c
#include <stdio.h>

int checkPrime(int);

int main()
{

    int lowerLimit, upperLimit;

    printf("Input lower limit: ");
    scanf("%d", &lowerLimit);

    printf("Input upper limit: ");
    scanf("%d", &upperLimit);

    int sum = 0;

    for (int i = lowerLimit + 1; i <= upperLimit; i++) {
        if(checkPrime(i))
            sum += i;
    }

    printf("Sum of prime numbers between %d-%d: %d\n", lowerLimit, upperLimit, sum);
    return 0;
}

int checkPrime(int num) {
    for (int i = 2; i <= num / 2; i++)
    {
            if (num % i == 0) {
                return 0;
                break;
            }
    }

    return 1;
}
